store/principal.go

```diff
@@ -262,9 +271,10 @@ func (s *Store) composePrincipal(ctx context.Context, raw *principalRaw) (*api.P
 				zap.Int("id", principal.ID),
 				zap.String("name", principal.Name),
 			)
-			return nil, errors.Wrapf(err, "member with PrincipalID %d not exist", principal.ID)
+			principal.Role = api.Developer
```

Comment:
The reason we separate principal from member role is that we reserve the design space that a bytebase deployment can contain multiple workspaces. So for a particular user, she may only belong to some of the workspaces.

Thus, we can't arbitrarily assign the developer role if membership is not found. 

System account is still a user account, it just happens that the user represents a service instead of   a human, so it would follow the existing RBAC model (the system account could be Owner, DBA, developer at the workspace level, and Owner / Developer at the project level)